tests/sentry/api/endpoints/test_project_rule_details.py

```diff
@@ -1552,3 +1558,44 @@ def test_simple(self):
         assert not Rule.objects.filter(
             id=self.rule.id, project=self.project, status=ObjectStatus.PENDING_DELETION
         ).exists()
+
+    @with_feature("organizations:workflow-engine-issue-alert-dual-write")
+    def test_dual_delete_workflow_engine(self):
```

Comment:
can we also test deleting with the feature flag if we don't have the corresponding models in workflow engine? like this exact test but without `migrate_issue_alert`